productivity-suite/app/legacy-app/src/worker/script.ts

```diff
@@ -173,4 +173,41 @@ gateway.registerFragment({
   },
 });
 
+gateway.registerFragment({
+  fragmentId: "news",
+  // Note: deployment part of the url is fine also for local development since then
+  //       only the path part of the url is being used
+  getBaseUrl: () =>
+    `https://productivity-suite-news-fragment.devdash.workers.dev`,
+  prePiercingStyles: `
+    :not(piercing-fragment-outlet) > piercing-fragment-host[fragment-id="news"] {
+      position: absolute;
+      top: 24.6rem;
```

Comment:
I'm sure this needs to be tweaked for various screensizes, it is a bit of a fidgety process, let me know if you'd like me to take care of it 🙂

Reply:
I didn't touch the CSS at all. I was trying to keep the example as is as much as possible. If you want to tweak it go to town.